ani-cli

```diff
@@ -395,8 +395,8 @@ open_episode () {
 		episode=$(printf "%03d" "$episode")
 		{
 		    #uncomment this below line if you are getting low download speeds, and comment next one after below line
```

Comment:
adapt the comment to paraphrase "if you have low bandwidth or low cpu power uncomment the line below"
-x 16 -s 16 is no joke

Reply:
i change the comment and put back the #

Reply:
this changes is not needed i will delete the file